kcidb/db/__init__.py

```diff
@@ -538,7 +538,22 @@ def __init__(self, *args, **kwargs):
             dest="test_ids",
             action='append',
         )
-
+        self.add_argument(
+            '-n', '--incident-id',
+            metavar="ID",
+            default=[],
+            help='ID of an incident to match',
+            dest="incident_ids",
+            action='append',
+        )
+        self.add_argument(
+            '-i', '--issue-id',
+            metavar="ID",
+            default=[],
+            help='ID of an issue to match',
+            dest="issue_ids",
+            action='append',
+        )
```

Comment:
Also, could you switch the order of those? "Issues" are parents of "incidents" in the schema, and the rest of these options are sorted by schema depth.

Reply:
okay